capstone-app/src/app/modules/history/wiki-media/wiki-media.component.scss

```diff
@@ -0,0 +1,33 @@
+.history {
+  background-color: #f5f5f5;
+  margin: 15px;
+}
+
+.history-title {
+  font-style: italic;
+  margin: 15px;
+}
+
+.history-body {
+  margin: 15px;
+}
+
+.more-reading {
+  display: inline-flex;
+  font-style: italic;
+  margin: 15px;
+}
+
+p {
+  margin: 15px;
+}
+
+a {
+  font-style: italic;
+}
+
+#more-reading-title {
+  font-weight: 500;
```

Comment:
Can you alphabetize this section of CSS because it seems the alphabetization is a little out of order. font-weight should go after font-variant.